db/migrate/20200602181502_add_blog_post_count_to_metric_name_type.rb

```diff
@@ -0,0 +1,16 @@
+class AddBlogPostCountToMetricNameType < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
I believe you should add `disable_ddl_transaction!` 
WDYT @mrubi-rootstrap ?

Reply:
If the migration fails with enums in Postgres that fixes it, yes

Reply:
Done! Thank you!